docker/templates/mariadb/tagged/Dockerfile.template

```diff
@@ -25,3 +25,7 @@ RUN set -x \
         && apt-get purge -y --auto-remove ca-certificates wget
 
 RUN echo "INSERT INTO kaui.kaui_allowed_users (kb_username, description, created_at, updated_at) values ('admin', 'super admin', NOW(), NOW());" > /docker-entrypoint-initdb.d/021_kaui_admin.sql
+
+RUN mkdir -p /tmp/replicate
+
+CMD ["mysqld", "--secure-file-priv=/tmp/replicate"]
```

Comment:
Could you set it to an empty string instead, to allow from everywhere (this was the default before MySQL 5.7.16)? That way, no need to change KPM (and no need to update the tests which are currently failing).

If not, could we set it to `/tmp` as it's going to be the default for `Dir.mktmpdir` in the Docker image?

Reply:
I tried the `--secure-file-priv=""` but got a failure at start time, so reverted to using `/tmp` 

```
replicate_db | 	command was: mysqld --secure-file-priv="" --verbose --help --log-bin-index=/tmp/tmp.upGHZ9hTL0
replicate_db | 	2021-07-28 17:44:18 0 [Warning] Failed to normalize the argument for --secure-file-priv.
```